meteor/.meteor/versions

```diff
@@ -98,7 +98,7 @@ templating-compiler@1.3.3
 templating-runtime@1.3.2
 templating-tools@1.1.2
 tracker@1.2.0
-twbs:bootstrap@3.3.6
+twbs:bootstrap@3.3.5
```

Comment:
@Phrozyn was this an intentional bootstrap downgrade?

Reply:
If you look at previous commits you'll see that jeff's commit is what reverted this to an earlier version of bootstrap. This was to facilitate a later version of jquery.

Reply:
ack.  Mostly just wanted to make sure intentional vs NPM mistake.